x/sequencer/keeper/funds.go

```diff
@@ -3,28 +3,31 @@ package keeper
 import (
 	errorsmod "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	commontypes "github.com/dymensionxyz/dymension/v3/x/common/types"
 	"github.com/dymensionxyz/dymension/v3/x/sequencer/types"
 )
 
-func (k Keeper) bondDenom(ctx sdk.Context) string {
-	return k.GetParams(ctx).MinBond.Denom
+func (k Keeper) minBond(ctx sdk.Context, rollapp string) sdk.Coin {
+	nDym := k.rollappKeeper.MinBond(ctx, rollapp)
+	ret := commontypes.DYMCoin
+	ret.Amount = ret.Amount.Mul(nDym)
+	return ret
 }
 
-func (k Keeper) validBondDenom(ctx sdk.Context, c sdk.Coin) error {
-	d := k.bondDenom(ctx)
-	if c.Denom != d {
-		return errorsmod.Wrapf(types.ErrInvalidDenom, "expect: %s", d)
+func validBondDenom(c sdk.Coin) error {
+	if c.Denom != types.BondDenom {
+		return errorsmod.Wrapf(types.ErrInvalidDenom, "expect: %s", types.BondDenom)
 	}
 	return nil
 }
 
-func (k Keeper) sufficientBond(ctx sdk.Context, c sdk.Coin) error {
-	if err := k.validBondDenom(ctx, c); err != nil {
+func (k Keeper) sufficientBond(ctx sdk.Context, rollapp string, c sdk.Coin) error {
```

Comment:
do we need some validation in regard of the kick threshold?
will be weird to have a kick threshold higher than the bond

Reply:
We just be careful
I'm not about to add cross module param verification